website/docs/components/rich-tooltip/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,215 @@
+## Usage
+
+### When to use
+
+- For adding extra information to a UI element that consists of more than a simple sentence, or structured content with a specific layout.
+- When it’s necessary to _temporarily persist_ extra information in the UI; e.g., if the user benefits from cross-referencing information in the component with other elements in the UI.
+
+!!! Info
+
+**Temporarily persist:** by this we mean the component will remain visible within UI until it is explicitly dismissed by the user, allowing the toggling open of the component and cross-referencing the content it contains with the rest of the UI. More details on this can be found in the [interaction](#interaction) section.
+!!!
+
+### When not to use
+
+- For extra information that consists of basic text, consider using the [Tooltip](/components/tooltip) instead.
+- For complex functions that change or alter the UI, consider a [Dropdown](/components/dropdown) instead.
+- As a double-confirmation for destructive actions, use a [Modal](/components/modal) instead.
+- If the information is **essential** to the user completing a task. In this case, consider placing this information within the page and associated with a specific UI element.
+
+### Rich Tooltip vs. Tooltip
+
+While there is overlap between the Rich Tooltip and the "standard" [Tooltip](/components/tooltip), which component to use depends on:
+
+- the complexity of the content
+- whether structural or layout elements are used
+- whether there are interactive elements; e.g., links or micro-interactions
+- whether the component should persist within the UI
+
+The _majority_ of use-cases can be categorized one of two ways:
+
+1. "Standard" Tooltip: simple, text-based content with support for rudimentary formatting (_italic_, **bold**, `monospace`, etc).
+2. Rich Tooltip: _relatively_ more complex, dynamic content with support for different layouts, organization of content, links, micro-interactions, diagrams, etc.
+
+View more detailed examples of the content the Rich Tooltip could contain in the [content](#content) section.
+
+## Placement
+
+A Rich Tooltip can be placed in various positions relative to the UI element it is associated with. Choose the placement based on the context and available space around the toggle.
+
+The most common options are: `top`, `bottom`, `left`, `right`.
+
+![Placement examples of the Rich Tooltip](/assets/components/rich-tooltip/rich-tooltip-placement.png)
+
+The Rich Tooltip supports additional placement options: `top-start`, `top-end`, `bottom-start`, `bottom-end`, `left-start`, `left-end`, `right-start`,and `right-end`.
+
+!!! Info
+
+The placement property is _relative_ to the element that toggles the component, rather than the direction of the pointer. E.g., `placement=left` will render the component to the _left_ of the toggle, but the pointer will be pointing to the right.
+!!!
+
+### Collision detection
```

Comment:
The `placement` is the initial location of the tooltip. It may even stay in that position if the user doesn't scroll or resize the page. Collision detection kicks in only in certain conditions. So designers/developers may want to set a preferred (initial) location for the tooltip, depending on where it's better it appears (eg to avoid overlapping other content).